src/main/kotlin/no/nav/syfo/service/MerVeiledningVarselService.kt

```diff
@@ -0,0 +1,50 @@
+package no.nav.syfo.service
+
+import no.nav.syfo.kafka.producers.dittsykefravaer.domain.DittSykefravaerMelding
+import no.nav.syfo.kafka.producers.dittsykefravaer.domain.OpprettMelding
+import no.nav.syfo.kafka.producers.dittsykefravaer.domain.Variant
+import no.nav.syfo.syketilfelle.SyketilfellebitService
+import org.slf4j.LoggerFactory
+import java.net.URL
+import java.time.ZoneOffset
+
+class MerVeiledningVarselService(
+    val senderFacade: SenderFacade,
+    val syketilfellebitService: SyketilfellebitService
+) {
+    private val log = LoggerFactory.getLogger("no.nav.syfo.service.MerVeiledningVarselService")
+
+    fun sendVarselOgOppgaveTilArbeidstaker(
+        fnr: String,
+        uuid: String,
+        varselContent: String,
+        varselUrl: URL
+    ) {
+        senderFacade.sendTilBrukernotifikasjoner(uuid, fnr, varselContent, varselUrl, null)
```

Comment:
Hvis vi skal følge tidligere pattern (MotebehovVarselService) så kan denne servicen selv bestemme hvilket content og url som skal sendes, i stedet for å få det inn som parametere